.script/utils/gitWrapper.ts

```diff
@@ -1,21 +1,35 @@
 import { cli, devOps } from "@azure/avocado";
 import * as logger from "./logger";
 import "./stringExtenssions";
+import { PullRequestProperties } from '@azure/avocado/dist/dev-ops';
 
-export async function GetDiffFiles(fileTypeSuffixes?: string[], filePathFolderPreffixes?: string[]) {
+var pullRequestDetails: PullRequestProperties | undefined;
+var isPullRequestDetailsInitialized: boolean = false;
+
+export async function GetPRDetails() {
+  if (!isPullRequestDetailsInitialized){
```

Comment:
Why need `isPullRequestDetailsInitialized`? same comment when call to this function., maybe we can call it once